crates/loader/src/cache.rs

```diff
@@ -130,12 +141,14 @@ impl Cache {
                 .with_context(|| format!("failed to create assets directory `{}`", p.display()))?;
         }
 
+        *self.dirs_ensured.write().await = true;
+
         Ok(())
     }
 }
 
 #[cfg(windows)]
-fn safe_name(digest: impl AsRef<str>) -> impl AsRef<Path> {
+fn safe_name(digest: impl AsRef<str>) -> impl AsRef<std::path::Path> {
```

Comment:
This change occurs because `Path` is now unused except for here.  (I could put a `cfg` on the module level `use` of it, but this seemed less intrusive.)